src/packet/signature.js

```diff
@@ -133,15 +131,27 @@ class SignaturePacket {
     this.signedHashValue = bytes.subarray(i, i + 2);
     i += 2;
 
-    this.signature = bytes.subarray(i, bytes.length);
+    this.signature = crypto.signature.parseSignatureParams(this.publicKeyAlgorithm, bytes.subarray(i, bytes.length));
```

Comment:
I would rename this property to `this.params`, if it's not too much work. `signature.signature` is a bit confusing (even though I know it was already the case).

Reply:
done